### PR TITLE
Created OskiBot.yml

### PR DESCRIPTION
**Pull request by michaelbzhu**
This is a project that I built for CalHacks 6.0, where I won the Cisco API and Transposit API awards. I want to continue it as a side project while I keep up with school.

I think it would be a great fit for Up for Grabs and a nice starter PR for beginners to familiarize themselves with open source.

Let me know if you have any questions or feedback!

### CODE REVIEW

File: _data/projects/OskiBot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: OskiBot
+desc: An all-purpose chatbot for UC Berkeley students built on Cisco's Webex Teams messaging application
+site: https://github.com/mbzhu1/oskibot
+tags:
+  - up-for-grabs
+  - hacktoberfest
+  - enhancement
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/mbzhu1/oskibot/issues/2
```
Comment (shiftkey) [inline _data/projects/OskiBot.yml]: This link should be to the issues labelled with `up-for-grabs`:

```suggestion
  link: https://github.com/mbzhu1/oskibot/labels/up-for-grabs
```

File: _data/projects/OskiBot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: OskiBot
+desc: An all-purpose chatbot for UC Berkeley students built on Cisco's Webex Teams messaging application
+site: https://github.com/mbzhu1/oskibot
+tags:
+  - up-for-grabs
```
Comment (shiftkey) [inline _data/projects/OskiBot.yml]: These links should help with describing the project, to help with searching. I'm going to add some suggestions based on skimming the project, but feel free to choose your own

```suggestion
  - chatbot
```

File: _data/projects/OskiBot.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  - hacktoberfest
```
Comment (shiftkey) [inline _data/projects/OskiBot.yml]:
```suggestion
  - node.js
```

File: _data/projects/OskiBot.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  - enhancement
```
Comment (shiftkey) [inline _data/projects/OskiBot.yml]:
```suggestion
  - azure
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
Thanks for submitting @mbzhu1! 

I added some feedback on things to improve, but this is feeling pretty close to landing after that.

**Comment by michaelbzhu** [comment]
Thank you @shiftkey! I have made the changes you requested. Please let me know if you have any more questions or feedback!

**Review by shiftkey — APPROVED** [review]
@mbzhu1 thanks for submitting!

**Pull request merged**